components/automate-ui/src/app/services/projects-filter/projects-filter.reducer.spec.ts

```diff
@@ -239,9 +239,14 @@ describe('projectsFilterReducer', () => {
         genUnassignedProject()
       );
 
-      it('displays "(unassigned)"', () => {
+      it('does not display filter', () => {
+        const { filterVisible } = projectsFilterReducer(initialState, action);
+        expect(filterVisible).toEqual(false);
```

Comment:
Suggest also add a check for this to be true in at least one case that is **not** just unassigned.

Reply:
I added it everywhere to be consistent with the testing suite. Feels a little redundant, but such is testing.